<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QVTKWidget" name="widget_VTK_1" native="true"/>
    </item>
    <item row="0" column="1">
     <widget class="QVTKWidget" name="widget_VTK_2" native="true"/>
    </item>
    <item row="0" column="2">
     <widget class="QVTKWidget" name="widget_VTK_3" native="true"/>
    </item>
    <item row="0" column="3">
     <widget class="QVTKWidget" name="widget_VTK_4" native="true"/>
    </item>
    <item row="1" column="0">
     <widget class="QVTKWidget" name="widget_VTK_5" native="true"/>
    </item>
    <item row="1" column="1">
     <widget class="QVTKWidget" name="widget_VTK_6" native="true"/>
    </item>
    <item row="1" column="2">
     <widget class="QVTKWidget" name="widget_VTK_7" native="true"/>
    </item>
    <item row="1" column="3">
     <widget class="QVTKWidget" name="widget_VTK_8" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1029</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header location="global">qvtkwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
